README.md

```diff
@@ -48,3 +51,11 @@ To stop isc-dhcp-server:
 `$ sudo killall dhcpd`
 
 Lease information is stored in /var/lib/dhcp/dhcpd.leases on linux, and /var/db/dhcpd.leases on OSX.
+
+## CI/testing
+
+The unit tests can be run with standard node tools:
+
+    npm test
+
+`./HWIMO-TEST` will run local tests, and was built for running on a jenkins build slave, and will run the tests, jshint, and code coverage all together.
```

Comment:
since it existed, I thought it worthwhile - it basically just runs the tests, same as Travis, but oriented towards a jenkins CI runner